bindings/python/py_src/safetensors/paddle.py

```diff
@@ -0,0 +1,141 @@
+import os
+from typing import Dict, Optional, Union
+
+import numpy as np
+
+import paddle
+from safetensors import numpy
+
+
+np.bool = bool
```

Comment:
What is this for ?

Reply:
i found this error. without  np.bool = bool,
![image](https://user-images.githubusercontent.com/50394665/217241205-6b6bb9a0-9032-4ac7-aae6-21721d5a7df8.png)

Reply:
```python
from safetensors.paddle import load_file, save_file
import paddle
import numpy as np
# numpy version 1.24.2
# np.bool = bool
file_path = "model.safetensors"
tensors = {"embedding": paddle.randint(0, 2, (512, 1024)).cast("bool"), "attention": paddle.zeros((256, 256))}
save_file(tensors, file_path)
loaded2 = load_file(file_path)
```